app/views/reports/regions/who_report.csv.erb

```diff
@@ -0,0 +1,103 @@
+<%
```

Comment:
I'm not sure if this being a view brings any value and would prefer it be in a ruby file, but I've followed the existing pattern.